### DISCUSSION TITLE
useModal 没有找到命名空间

### DISCUSSION THREAD

**Discussion by Atw-Lee** [umijs/umi#11236] [Q&A] [answered]
按照文档使用useModal，ts报错提示没有找到Namespaces，要怎么定义呢

**Comment by fz6m** [comment, marked as answer]
umi 4 本身不包含 model 功能，需要安装 `@umijs/plugins` 之后配置使用独立的某个功能：

```ts
export default {
  plugins: ['@umijs/plugins/dist/model'],
  model: {}
}
```

或者直接使用 max ，默认包含了 model 。

**Reply by Atw-Lee** [reply]
感谢🙏。找到原因了，我要关了vscode重开，裂开。
